core/rfb.js

```diff
@@ -2,12 +2,11 @@
  * noVNC: HTML5 VNC client
  * Copyright (C) 2012 Joel Martin
  * Copyright (C) 2018 Samuel Mannehed for Cendio AB
+ * Copyright (C) 2018 Pierre Ossman for Cendio AB
```

Comment:
If people are ok with it, I think we should just switch this over to `The noVNC authors` -- less copyright churn, among other things.  A number of other opensource projects do this.

Reply:
I'm fine with that. When I was the main author with a few people sending bug fixes it was fine just to have my name, but at this point it's a team project so we should reflect that. In the U.S. at least, an author has copyright regardless of whether they label it as such so it's more for documentation than any legal reasons.

Reply:
#674